canonical-web-and-design/dqlite.io
# Add discourse docs module

## Done
Add discourse docs module to load content from the [discourse docs category[(https://discourse.dqlite.io/t/dqlite-documentation/21).

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8037/docs
- Check it renders correctly
- Click on the pages and check they work too

## Issue / Card

Fixes #9
